src/elements/TabbedContainer/TabbedContainer.js

```diff
@@ -0,0 +1,99 @@
+import './TabbedContainer.less';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import List from '../List/List';
+
+class TabbedContainer extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			// set default showing tab (0 = first tab, 1 = second tab, etc...)
+			activeIndex: 0
+		};
+
+		this.changeTabs = this.changeTabs.bind(this);
+		this.renderNavItems = this.renderNavItems.bind(this);
+		this.renderOneNavItem = this.renderOneNavItem.bind(this);
+	}
+
+	changeTabs(event, key) {
+		event.preventDefault();
+
+		this.setState({
+			activeIndex: key
+		});
+	}
+
+	renderNavItems(key) {
+		const tab = this.props.children[key] ? this.props.children[key] : null,
+			keyNum = Number(key);
+
+		const navItemClasses = classNames(
+			this.state.activeIndex === keyNum ? 'active' : ''
+		);
+
+		return (
+			<li key={key} className={navItemClasses}>
+				<a href="#" onClick={ (e) => this.changeTabs(e, keyNum) }>{ tab.props.title }</a>
+			</li>
+		);
+	}
+
+	renderOneNavItem() {
+		const oneTab = React.Children.only(this.props.children);
+
+		return (
+			<li className="active">
+				<a href="#">{ oneTab.props.title }</a>
+			</li>
+		);
+	}
+
+	render() {
+		let index = 0,
+			renderNav;
+		const isActive = this.state.activeIndex;
+
+		const tabs = React.Children.map(this.props.children, (child) => {
+			return React.cloneElement(child, {
+				active: child.props.active === true ? true : (isActive === index++)
```

Comment:
Few things on this line:
- Booleans should never be compared (ie `child.props.active === true ? ....`). Simply using a boolean value in a selector or ternary statement is the preferred method `child.props.active ? ...`

- When returning the same value that is being checked for truthiness in a ternary, it's preferred to switch to a selector statement instead: `child.props.active || ....<value to be returned if previous is falsy>`

- Checking the active index and incrementing the index counter can be confusing. Although it would be more verbose, if you need to manually increment a counter this is best done on a separate line.